apps/nextjs-website/src/lib/types/webinar.ts

```diff
@@ -5,8 +5,8 @@ export type Webinar = {
   readonly description: string;
   readonly html?: string;
   readonly speakers: readonly Speaker[];
-  readonly startDateTime?: Date;
-  readonly endDateTime?: Date;
+  readonly startDateTime?: string;
```

Comment:
Write a comment about why this should be a string.

Reply:
@datalek In the issue description you can find the reason for these changes.
Do you want me to add the same comment in the code too?

Reply:
@jeremygordillo here you should only explain (the repo is public, but the board is not) why you converted a Date to a string in the Webinar type.
I suppose that on the one hand, you expect that the API will send a string, and on the other hand, converting to date only in the component you resolve the problem of the non-corresponding dates, am I right?

Reply:
Yes, you're right. I have improved the PR description with the explanation of these changes.